src/Components/Users/UsersForm.js

```diff
@@ -1,38 +1,78 @@
-import React, { useState } from 'react';
-import '../FormStyles.css';
+import React, { useState, useEffect } from "react";
+import "../FormStyles.css";
+import UsersFormTerms from "./UsersFormTerms";
 
 const UserForm = () => {
-    const [initialValues, setInitialValues] = useState({
-        name: '',
-        email: '',
-        roleId: ''
-    })
+  const [initialValues, setInitialValues] = useState({
+    name: "",
+    email: "",
+    roleId: "",
+  });
+  const [acceptTerms, setAcceptTerms] = useState(false);
+  const [sendButton, setSendButton] = useState(true);
 
-    const handleChange = (e) => {
-        if(e.target.name === 'name'){
-            setInitialValues({...initialValues, name: e.target.value})
-        } if(e.target.name === 'email'){
-            setInitialValues({...initialValues, email: e.target.value})
-        }
-    }
+  useEffect(() => {
+    acceptTerms && setSendButton(false);
+  }, [acceptTerms]);
+
+  const showTerms =
+    initialValues.name.length > 0 &&
+    initialValues.email.length > 0 &&
+    initialValues.roleId.length > 0;
 
-    const handleSubmit = (e) => {
-        e.preventDefault();
-        console.log(initialValues);
+  const handleChange = (e) => {
+    if (e.target.name === "name") {
```

Comment:
maybe you can improve this and instead of using multiple if you can use a switch